src/dcommands/timer.js

```diff
@@ -31,10 +31,8 @@ class TimerCommand extends BaseCommand {
     }
 
     async event(args) {
-        let duration = moment.duration(moment() - moment(args.starttime));
-        duration.subtract(duration * 2);
         bu.send(args.channel, {
-            content: `:alarm_clock: *Bzzt!* <@${args.user}>, the timer you set ${duration.humanize(true)} has gone off! *Bzzt!* :alarm_clock:`,
+            content: `:alarm_clock: *Bzzt!* <@${args.user}>, the timer you set <t:${moment(args.starttime).unix()}:R> has gone off! *Bzzt!* :alarm_clock:`,
```

Comment:
That'd be the same comment as for above changes: would it be possible to keep the value calculation outside of the string interpolation?

Thanks a lot!

Reply:
Sorry for late response, wasn't able to see these reviews before, as I was at phone :P
Anyways I did what u asked for, pls checkout commits below. Also suggest me about [this.](https://github.com/blargbot/blargbot/pull/275#issuecomment-896046169)

Reply:
Would it be possible to use a variable wherever you need to display something that has to be calculcated/instanciated before displayed? 
Here's an example: 

> ![image](https://user-images.githubusercontent.com/32441291/129190740-08eae696-81b8-452d-afc1-9c8719298698.png)

Can you make the changes please?